src/PrestaShopBundle/Resources/views/Admin/Product/form_combinations_bulk.html.twig

```diff
@@ -0,0 +1,70 @@
+{{ form_start(form, {'action': '#', 'method': 'GET'}) }}
+<div class="col-md-12">
+  <div class="col-md-2">
```

Comment:
wow, I don't want to create a complete tree of div only to re order few blocks !
Do we have alternative "clean" solution ?